### DISCUSSION TITLE
api conection(foreign key array)

### DISCUSSION THREAD

**Discussion by MathewCognitive** [supabase/supabase#8099] [Questions] [answered]
how can i connect multiple tables in one api without using foreign key relation
have to add tags array field

**Comment by steve-chavez** [comment, marked as answer]
For now you'd need a junction table instead of an array to create a many-to-many relationship(see [this previous discussion](https://github.com/supabase/supabase/discussions/535)).

The new [computed relationships](https://github.com/PostgREST/postgrest/issues/2144)(docs being worked on) will allow you to create relationships(only for the SDKs/API) without foreign keys for these cases.
